library/DynamicReplicatedTableEntity.cs

```diff
@@ -43,61 +43,66 @@ public InitDynamicReplicatedTableEntity(string partitionKey, string rowKey)
         {
         }
 
-        public InitDynamicReplicatedTableEntity(string partitionKey, string rowKey, string etag, IDictionary<string, EntityProperty> properties)
+        public InitDynamicReplicatedTableEntity(string partitionKey, string rowKey, string etag, IDictionary<string, object> properties)
             : base(partitionKey, rowKey, etag, properties)
         {
         }
 
-        public override void ReadEntity(IDictionary<string, EntityProperty> properties, OperationContext operationContext)
+        public InitDynamicReplicatedTableEntity(TableEntity entity)
```

Comment:
Looks like you have used this API instead of one in line 46 in ReplicatedTable.cs, looks like for the previous API was using datetime as min value.. please see if it will have any impact

Reply:
This shouldn't have impact. Based on documentation, setting of Timestamp field will be ignored anyway
![image](https://github.com/Azure/rtable/assets/122410443/7ea8e1b8-7aeb-4964-9551-75fee0d42fcc)
https://learn.microsoft.com/en-us/dotnet/api/azure.data.tables.tableentity.timestamp?view=azure-dotnet